go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	k8s.io/apiserver v0.26.4
 	k8s.io/client-go v0.26.4
 	k8s.io/component-base v0.26.4
+	k8s.io/klog v1.0.0
```

Comment:
Why is this added? Usually we use `k8s.io/klog/v2`

Reply:
Sorry, accidentally added that. Removed.